### PR TITLE
PS-269: Fix `backup_locks_binlog` MTR tests (8.0)

### PR DESCRIPTION
**Pull request by inikep**
1. Add missing `--connection` that were removed along with `LOCK BINLOG FOR BACKUP`.
2. Re-record `main.backup_locks_binlog` MTR tests.

### CODE REVIEW

File: mysql-test/include/backup_locks_binlog.inc [hunk 1]
```diff
@@ -258,8 +258,14 @@ INSERT INTO tt_csv VALUES(5);
...
 
+--connection default
+--echo # connection default
+
 UNLOCK TABLES;
 
+--connection con1
+--echo # connection con1
+
 COMMIT;
```
Comment (laurynas-biveinis) [inline mysql-test/include/backup_locks_binlog.inc]: Does this COMMIT make sense in con1? I don't see an open transaction there

Comment (laurynas-biveinis) [inline mysql-test/include/backup_locks_binlog.inc]: Does this COMMIT make sense in con1? I don't see an open transaction there

Comment (laurynas-biveinis) [inline mysql-test/include/backup_locks_binlog.inc]: Is this COMMIT still required in con1? There does not seem to be an open transaction there

Comment (laurynas-biveinis) [inline mysql-test/include/backup_locks_binlog.inc]: Is this COMMIT still required in con1? There does not seem to be an open transaction there

**Comment by inikep** [comment]
PR was updated

**Review by laurynas-biveinis — APPROVED** [review]

**Pull request merged**